apps/ops-func/src/adapters/functions/split-delete-messages.ts

```diff
@@ -0,0 +1,29 @@
+import { StorageBlobHandler } from "@azure/functions";
+import { QueueClient } from "@azure/storage-queue";
+
+export const splitDeleteMessage =
+  (queueClient: QueueClient): StorageBlobHandler =>
+  async (blob, context) => {
+    if (!Buffer.isBuffer(blob)) {
+      context.error("Invalid input blob file");
+      return;
+    }
+    const lines = blob.toString("utf-8").trim().split("\n");
+
+    lines.forEach((line) => {
+      const [fiscalCode, messageId] = line.split(",");
+
+      if (!fiscalCode || !messageId) {
```

Comment:
you should also parse these values to make sure that they follow the correct format

Reply:
As we discussed [here](https://github.com/pagopa/io-messages/pull/181#discussion_r2042247750) I think it is better to not check that the format is correct